README.md

```diff
@@ -1,2 +1,429 @@
-# iceberg-catalog-migrator
-CLI tool to bulk migrate the tables from one catalog another without a data copy
+# Objective 
+Introduce a command-line tool that enables bulk migration of Iceberg tables from one catalog to another without the need to copy the data.
+
+There are various reasons why users may want to move their Iceberg tables to a different catalog. For instance,
+* They were using hadoop catalog and later realized that it is not production recommended. So, they want to move tables to other production ready catalogs.
+* They just heard about the awesome Arctic catalog (or Nessie) and want to move their existing iceberg tables to Dremio Arctic.
+* They had an on-premise Hive catalog, but want to move tables to a cloud-based catalog as part of their cloud migration strategy.
+
+Previously, before the Iceberg `1.1.0` release, the only way to migrate tables was by copying the data using the command `insert into targetCatalog.db.tableName as select * from sourceCatalog.db.tableName`.
+After the iceberg `1.1.0` release, all Iceberg Catalogs supports register table with the `catalog#registerTable()` API.
+However, custom code is needed to migrate all the tables in bulk. 
+**Hence, we introduce a CLI tool to migrate Iceberg tables in bulk from one Iceberg Catalog to another without a data copy.**
+
+# Iceberg-catalog-migrator
+Need to have java installed in your machine(JDK11 or later version) to use this CLI tool.
+
+Below is the CLI syntax:
+```
+$ java -jar iceberg-catalog-migrator-cli-0.1.0-SNAPSHOT.jar -h        
```

Comment:
Make sure that all code samples reflect the state of the code

Reply:
updated.